src/sagas/wallet.js

```diff
@@ -33,6 +33,7 @@ import { eventChannel } from 'redux-saga';
 import { getUniqueId } from 'react-native-device-info';
 import { t } from 'ttag';
 import { get } from 'lodash';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
What's this?

Reply:
A lack of attention. I removed this unused import in [9f3ff55](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/9f3ff558164e3d6ca930ad3e54256886e5c6fb69).